repos/system_upgrade/el7toel8/actors/persistentnetnamesdisable/actor.py

```diff
@@ -38,7 +39,7 @@ def process(self):
 
         if self.single_eth0(interfaces):
             self.disable_persistent_naming()
-        elif self.ethX_count(interfaces) > 1:
+        elif self.ethX_count(interfaces) > 0 and len(interfaces) > 1:
```

Comment:
Again, pretty minor and no blocking, but checking the length of interfaces is cheaper than searching for ethX, so I would change the order of checks.

Reply:
Good point. Just from my point of view, this looks more readable. But I will swap them